INSTRUMENTS/Serial/Serial/SERIAL.py

```diff
@@ -0,0 +1,95 @@
+from flojoy import flojoy, DataContainer
+from time import time, sleep
+import serial
+import numpy as np
+from datetime import datetime
+import plotly.express as px
+
+@flojoy
+def SERIAL(v, params):
+    '''
+    Node to take simple time dependent 1d data from an Ardunio, or a similar serial device.
+    For example you can record temperature following this tutorial:
+
+    https://learn.adafruit.com/thermistor/using-a-thermistor
+
+    with Serial.println(steinhart) as the only line printing.
+
+    It is important that the last line Arduino is returning is the data with a new line at the end (i.e. println()).
```

Comment:
exceeds the PEP-8 max line length
https://stackoverflow.com/questions/88942/why-does-pep-8-specify-a-maximum-line-length-of-79-characters

please install a PEP-8 linter

Reply:
PEP-8 changes made.